charts/spdk-csi/latest/spdk-csi/values.yaml

```diff
@@ -136,10 +148,14 @@ cachingnode:
 storagenode:
   create: false
   ifname: eth0
+  distr_ndcs: "1"
```

Comment:
We should help users and rename the publicly available properties to "numDataChunks" and "numParityChunks". Nobody external will understand ndcs and npcs.